codenames-front-end/src/pages/Authentication/SignUp/SignUpStyles.ts

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+
+export const StyledSingUp = styled.div`
```

Comment:
`StyledSignUp`

Reply:
I apologize for any problems with grammar